### PR TITLE
HPCC-16396 Enable latedef linting

### PR DESCRIPTION
**Pull request by GordonSmith**
Inline functions being called before they are declared will break when the JS is minimized.

Fixes HPCC-16396
Fixes HPCC-16293

Signed-off-by: Gordon Smith gordonjsmith@gmail.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq Please Review

File: esp/src/eclwatch/TopologyWidget.js [hunk 1]
```diff
@@ -131,10 +129,8 @@ define([
                                 [TargetClusterCount]: 1000,
                                 AutoRefresh:filter.AutoRefresh,
                                 MemThreshold:filter.MemThreshold,
-                                DiskThreshold:filter.DiskThreshold,
```
Comment (GordonSmith) [inline esp/src/eclwatch/TopologyWidget.js]: They were included twice (expand the source and you will see them)

File: esp/src/eclwatch/XrefDirectoriesWidget.js [hunk 2]
```diff
@@ -118,7 +118,6 @@ define([
                    newRows.push({
                         Name: row.Name,
                         Num: row.Num,
-                        MinSize: row.MinSize,
```
Comment (GordonSmith) [inline esp/src/eclwatch/XrefDirectoriesWidget.js]: It was included twice (expand the source and you will see it)

File: esp/src/eclwatch/PreflightDetailsWidget.js [hunk 3]
```diff
@@ -117,22 +117,16 @@ define([
             switch (row) {
                 case 1:
                     return this.i18n.Normal;
-                    break;
```
Comment (GordonSmith) [inline esp/src/eclwatch/PreflightDetailsWidget.js]: Unreachable code, due to return

File: esp/src/eclwatch/TopologyWidget.js (hunk 1 above)
Comment (miguelvazq) [inline esp/src/eclwatch/TopologyWidget.js]: Viewed source. You are correct.

File: esp/src/eclwatch/PreflightDetailsWidget.js (hunk 3 above)
Comment (miguelvazq) [inline esp/src/eclwatch/PreflightDetailsWidget.js]: Agreed.

**Comment by miguelvazq** [comment]
Looks good.

**Pull request merged**